notebooks/kubeflow_pipelines/pipelines/labs/kfp_pipeline.ipynb

```diff
@@ -347,7 +347,7 @@
     "Update  the below constants  with the settings reflecting your lab environment. \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ARTIFACT_STORE_URI = 'gs://hostedkfp-default-e8c59nl4zo'
</pre><p>could you update this example as well to illustrate the  change format? (i.e. copy from solutions)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>